nf_core/pipeline-template/lib/WorkflowMain.groovy

```diff
@@ -20,43 +20,14 @@ class WorkflowMain {
             "  https://github.com/${workflow.manifest.name}/blob/master/CITATIONS.md"
     }
 
-    //
-    // Generate help string
-    //
-    public static String help(workflow, params) {
-        {% if igenomes -%}
-        def command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome GRCh37 -profile docker"
-        {% else -%}
-        def command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --fasta reference.fa -profile docker"
-        {% endif -%}
-        def help_string = ''
-        help_string += NfcoreTemplate.logo(workflow, params.monochrome_logs)
-        help_string += NfcoreSchema.paramsHelp(workflow, params, command)
-        help_string += '\n' + citation(workflow) + '\n'
-        help_string += NfcoreTemplate.dashedLine(params.monochrome_logs)
-        return help_string
-    }
-
-    //
-    // Generate parameter summary log string
-    //
-    public static String paramsSummaryLog(workflow, params) {
-        def summary_log = ''
-        summary_log += NfcoreTemplate.logo(workflow, params.monochrome_logs)
-        summary_log += NfcoreSchema.paramsSummaryLog(workflow, params)
-        summary_log += '\n' + citation(workflow) + '\n'
-        summary_log += NfcoreTemplate.dashedLine(params.monochrome_logs)
-        return summary_log
-    }
 
     //
     // Validate parameters and print summary to screen
     //
     public static void initialise(workflow, params, log) {
         // Print help to screen if required
         if (params.help) {
-            log.info help(workflow, params)
-            System.exit(0)
+            return
```

Comment:
does groovy have empty `return`s?

Reply:
It doesn't. Actually, the whole if is not needed, thanks!